.eslintrc.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+	extends: [
+		'@nextcloud/eslint-config/typescript',
+	],
+	settings: {
+		jsdoc: {
+			mode: 'typescript',
```

Comment:
Is this not included in our eslint config as an override?

Reply:
So no, it's not part of the global eslint config :(

Reply:
SOmething to fix in another place, I removed that unused section